Source/CesiumRuntime/CesiumRuntime.Build.cs

```diff
@@ -110,45 +111,45 @@ public CesiumRuntime(ReadOnlyTargetRules Target) : base(Target)
             "uriparser",
             "webpdecoder",
             "ktx_read",
-        };
+    };
 
-        // Use our own copy of MikkTSpace on Android.
-        if (Target.Platform == UnrealTargetPlatform.Android || Target.Platform == UnrealTargetPlatform.IOS)
-        {
-            libs = libs.Concat(new string[] { "MikkTSpace" }).ToArray();
-            PrivateIncludePaths.Add(Path.Combine(ModuleDirectory, "../ThirdParty/include/mikktspace"));
-        }
+    // Use our own copy of MikkTSpace on Android.
+    if (Target.Platform == UnrealTargetPlatform.Android || Target.Platform == UnrealTargetPlatform.IOS)
+    {
+      libs = libs.Concat(new string[] { "MikkTSpace" }).ToArray();
+      PrivateIncludePaths.Add(Path.Combine(ModuleDirectory, "../ThirdParty/include/mikktspace"));
+    }
 
-        if (Target.Platform == UnrealTargetPlatform.Win64)
-        {
-            libs = libs.Concat(new string[] { "tidy_static", "zlibstatic" }).ToArray();
-        }
-        else
-        {
-            libs = libs.Concat(new string[] { "tidy", "z" }).ToArray();
-        }
+    if (Target.Platform == UnrealTargetPlatform.Win64)
+    {
+      libs = libs.Concat(new string[] { "tidy_static", "zlibstatic-ng" }).ToArray();
```

Comment:
This is the only real change I made; the rest was formatting. (Can undo it if old formatting was preferred)